.github/workflows/build_documentation.yml

```diff
@@ -46,7 +58,7 @@ jobs:
         run: |
           sudo apt-get -y update && sudo apt-get install -y libsndfile1-dev
 
-          pip install git+https://github.com/huggingface/doc-builder
+          pip install git+https://github.com/huggingface/doc-builder@kit
```

Comment:
note: merge after https://github.com/huggingface/doc-builder/pull/94 is merged & remove @kit